src/lib/updates/updateFeedEvents.js

```diff
@@ -0,0 +1,50 @@
+import { get } from 'lodash'
+import userStorage from '../gundb/UserStorage'
+
+const fromDate = new Date('2021/06/17')
+
+/**
+ * fix broken feed items
+ * @returns {Promise<void>}
+ */
+const updateFeedEvents = async (lastUpdate, prevVersion, log) => {
```

Comment:
this is not required. in anycase triggering _syncwithblockchain should be enough

Reply:
you mean syncTxWithBlockchain right?
from which startBlock?